packages/server/src/middleware/authorized.js

```diff
@@ -42,6 +42,10 @@ module.exports = (permName, getItemId) => async (ctx, next) => {
     ctx.throw(403, "User not found")
   }
 
+  if (ctx.user.accessLevel._id === ADMIN_LEVEL_ID) {
```

Comment:
This shouldn't be (i see it was already like this!)

This means that all End Users can do anything - change tables, remove screens etc etc